src/DataViews/DataView.cpp

```diff
@@ -243,12 +245,14 @@ void DataView::OnDisableFrameTrackRequested(const std::vector<int>& selection) {
       orbit_metrics_uploader::OrbitLogEvent::ORBIT_FRAME_TRACK_DISABLE_CLICKED);
 
   for (int i : selection) {
-    const FunctionInfo& function = *GetFunctionInfoFromRow(i);
+    const FunctionInfo* function = GetFunctionInfoFromRow(i);
+    if (function == nullptr) continue;
```

Comment:
As we disable the enable / disable frame track actions for the manually instrumented functions, so it might be better if we add a `ORBIT_CHECK(function!=nullptr)` here. This `OnEnableFrameTrackRequested` and `OnDisabelFrameTrackRequested` should not be invoked for the manually instrumented functions now.

Reply:
It's still invoked when both manually and dynamically instrumented entries are selected, right-clicked and "enable frame pointer" is clicked.